### PR TITLE
Support custom dashboard link

### PR DESCRIPTION
**Pull request by ka3de**
This PR adds a new configuration parameter `dashboard_link` in order to customize the dashboard link present inside the security tab in the Vulcan's UI main page. 
If set, the user will be redirected to the defined custom link defining the team's SDRN as a parameter in the request query string. Example:
```
redirect <dashboard_link>?team=<team_sdrn>
```
If the `dashboard_link` parameter is not set or is set to `/`, the dashboard link will lead to the previous [dashboard.html](https://github.com/adevinta/vulcan-ui/blob/master/src/dashboard.html) page.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -106,7 +114,7 @@ async function loadData(client, config) {
 function showTeams(teams) {
     teams = teams.sort((t1, t2) => t1.name.localeCompare(t2.name))
     teams.forEach(team => {
-        $('#teamsList').append(`<option value="${team.id}"> 
+        $('#teamsList').append(`<option value="${team.id}" tag="${team.tag}">
```
Comment (gguillemas) [inline src/index.js:117]: Although in HTML5 you can made up attributes, I think we should preface them with `data-` following the [spec](https://www.w3schools.com/tags/att_global_data.asp) to make it clear that they are for our internal usage.

**Review by gguillemas — DISMISSED** [review]
The change looks good to me! I added a small nitpick.

File: src/index.js (hunk 1 above)
Comment (ka3de) [reply src/index.js]: Got it! Didn't know about this. Just changed it.

**Review by gguillemas — APPROVED** [review]
LGTM

**Pull request merged**